tutorials/plot_creating_data_structures.py

```diff
@@ -76,9 +76,9 @@
 # ---------------------------------------------
 #
 # To create a :class:`mne.io.Raw` object from scratch, you can use the
-# :class:`mne.RawArray` class, which implements raw data that is backed by a
+# :class:`mne.io.RawArray` class, which implements raw data that is backed by a
 # numpy array.  Its constructor simply takes the data matrix and
-# :class:`mne.io.meas_info.Info` object:
+# :class:`mne.io.Info` object:
```

Comment:
Maybe this one should actually be top-level, too. It's not really *only* for raw IO, which is how we've been conceptualizing the `mne.io` namespace.